### PR TITLE
completed html-tables exercise, avallon smith 

### PR DESCRIPTION
**Pull request by avallonsmith**
image

Not too sure why everything is centered (it did it automatically). Also, please don't mind the random names, I was just having some fun xD

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <tr>
+          <th>Student Name</th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <th>Grace Hopper</th>
```
Comment (Zhiwutian) [inline html-tables/index.html:23]: you want to use the `th` elements in the `thead` but these ones should be `td` elements

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by avallonsmith** [comment]
image
image

Got it, my bad!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**